example/todo/todo.hpp

```diff
@@ -0,0 +1,39 @@
+//
+// lager - library for functional interactive c++ programs
+// Copyright (C) 2017 Juan Pedro Bolivar Puente
+//
+// This file is part of lager.
+//
+// lager is free software: you can redistribute it and/or modify
+// it under the terms of the MIT License, as detailed in the LICENSE
+// file located at the root of this source code distribution,
+// or here: <https://github.com/arximboldi/lager/blob/master/LICENSE>
+//
+
+#include <immer/flex_vector.hpp>
```

Comment:
This can be removed, and you should add `<string>` and `<vector>`

Reply:
Good catch :) I actually wanna use `flex_vector`, but need to tweak the `[]` operator of the cursors first.